src/validations/MeasureSchemaValidator.ts

```diff
@@ -17,7 +17,8 @@ export const MeasureSchemaValidator = Yup.object().shape({
     .matches(
       /^[A-Z][a-zA-Z0-9]*$/,
       "Measure library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters."
-    ),
+    )
+    .max(64, "Measure Library name cannot be more than 64 characters."),
```

Comment:
According to AC message would be: **Library name cannot be more than 64 characters.**